.vscode/settings.json

```diff
@@ -9,20 +8,11 @@
         "editor.defaultFormatter": "esbenp.prettier-vscode",
         "editor.formatOnSave": true
     },
-
     "eslint.alwaysShowStatus": true,
-    "eslint.autoFixOnSave": true,
     "eslint.lintTask.enable": true,
     "eslint.codeAction.showDocumentation": {
         "enable": true
     },
-    "eslint.validate": [
-        "javascript",
-        {
-            "autoFix": true,
-            "language": "typescript"
-        },
-    ],
```

Comment:
Is this deprecated?

Reply:
![Screen Shot 2020-01-29 at 5 47 58 PM](https://user-images.githubusercontent.com/4524823/73404461-8ffb0180-42bf-11ea-8c08-7581656a91fb.png)